test/endpoint_tests/test_graphql.py

```diff
@@ -0,0 +1,153 @@
+from boxwise_flask.db import db
+from boxwise_flask.models import Camps, Cms_Users
+
+
+def test_all_bases(client):
+    """example test for graphql endpoint"""
+
+    db.connect_db()
+    Camps.create(id=1, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=2, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=3, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=4, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=5, organisation_id=1, name="some text1", currencyname="hello")
+    db.close_db(None)
+
+    graph_ql_query_string = """query { \
```

Comment:
In the test after there are no backslashes so I'd remove them here as well, for consistency and because it's less 'noisy' visually

Edit: also for the occurences below

Reply:
I noticed this in the latest bit of work I've done. You are correct!